lib/http2.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const { HttpChannel } = require('./http.js');
+
+const HEADERS = {
+  'X-XSS-Protection': '1; mode=block',
+  'X-Content-Type-Options': 'nosniff',
+  'Strict-Transport-Security': 'max-age=31536000; includeSubdomains; preload',
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Methods': 'POST, GET, OPTIONS',
+  'Access-Control-Allow-Headers': 'Content-Type',
+};
+
+const MIME_TYPES = {
```

Comment:
We have both collections in `http.js` and it will be better to extract them to separate file `http-constants.js` or so